TimJohnson/Homework_Week1.js

```diff
@@ -0,0 +1,139 @@
+1a)	"1" == 1; 	true;  using == means equal to and will cast the 1 on the right side of the equation as a string
+1b)	"1" === 1;  false;  using === means that the comparison includes not only equal value but equal type.  (here is string versus number)
+1c)	 x == 'x';	false;  string 'x' is converted tonumber(x) which is different than the number x
+1d)	 x == (x+''); true;  number is equal to that number + an empty string
+1e)	'' == ' ';	false;  empty sting is always false but ' ' is a valid string
+1f)	x = true;	true;	assignment variable can assign x = true
+1g)	var x; x == 'undefined';	false;	undefined is equal to null but not a true value
+1h)	'9'<'10'; false;	These both evaluate to a string so they both are equal
+1i)	typeof x + 1 === "number";	false;	typeof x + 1 converts typeof x to a string with the type and adds 1 (e.g. 'number1') which is not = 'number'
+1j)	typeof x % 2 === "number";	false;	typeof x % 2 = NaN which is not equal in value to the string 'number'
+1k)	typeof (x % 2) === "number";	true;  typeof (x % 2) = 'number' which is equal to the string "number"
+1l)	x++ == ++x;	false;  these are valid increment operators where one is post decrement and the second is pre-decrement which results in different values for x
+1m)	++x == x++;	true;	the pre-increment and post increment values are not implemented until after x is compared to x
+1n)	"1"+x == 1+x;	false; the left side evaluates to '1x' as a string whereas the right side evaluates to x+1. e.g. if x where 3 then left side = '13' and right = 4
```

Comment:
Remember that in this case x is not 'x'. The left side will not evaluate to '1x' but rather 1 + whatever x contains as a value. Try setting x to 4, or 'grapes' and running the code again.